test/lazy/test_batch_repeat_lazy_tensor.py

```diff
@@ -23,6 +26,7 @@ def evaluate_lazy_tensor(self, lazy_tensor):
 
 class TestBatchRepeatLazyTensorBatch(BatchLazyTensorTestCase, unittest.TestCase):
     seed = 0
+    should_test_sample = True
 
     def create_lazy_tensor(self):
         toeplitz_column = torch.tensor([[4, 0, 0, 1], [3, 0, -0.5, -1]], dtype=torch.float, requires_grad=True)
```

Comment:
Right now I don't think `root_decomposition` and `root_inv_decomposition` are tested in `LazyTensorTestCase`. It would probably be good to add those in.